src/pages/home/report/ReportActionItem.tsx

```diff
@@ -178,7 +179,7 @@ function ReportActionItem({
     );
     const theme = useTheme();
     const styles = useThemeStyles();
-    const [reportNameValuePairs] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${report?.reportID ?? -1}`);
+    const [reportNameValuePairs] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${!isEmpty(report?.reportID) ? report?.reportID : -1}`);
```

Comment:
Oh yeah `|| -1` should be enough, I tried too much to make it work without disabling an eslint rule 😄 This change is required because when clearing the state with an open report, at some point we are receiving an empty report object where `reportID` is an empty string. It was causing the entire app crashing, since it was trying to access `reportNameValuePairs_` key. I added a comment explaining it together with disabled rule